src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.datajpa.repository;
+
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+    public Category getCategoryByName(String name);
```

Comment:
try this method name:
```suggestion
    public Category findByName(String name);
```

Reply:
+ `name` column is unique?

Reply:
I guess it should be, because it doesn't make sense to have two categories that are the same